### ISSUE TITLE
bug: css-declaration-sorter orders `column-gap` above `gap`

### ISSUE THREAD

**Issue by ludofischer** [Siilwyn/css-declaration-sorter#380]
Save this style in `input.css`:
```css
.card {
   gap: 0;
   column-gap: 10px;
   row-gap: 15px;
}
```
Install `postcss`, `css-declaration-sorter` and `postcss-cli`.
Run
`postcss input.css --use css-declaration-sorter | cat`
The result is:
```css
.card {
   column-gap: 10px;
   gap: 0;
   row-gap: 15px;
}
```
which is wrong because `gap` now overrides ` column-gap`. Instead `css-declaration-sorter` should preserve the order of `gap` and `column-gap`.

**Comment by ludofischer** [comment]
I've tried to update the properties by running updating `@mdn/browser-compat-data` and running `npm run scrape` but the `scrape` script just crashes.

**Comment by Siilwyn** [comment]
Sorry looks like I missed the `gap` shorthand data when it was included.
Regarding the scrape script crashing, seems like something changed in either the package or `--experimental-import-meta-resolve` flag behaviour, will look into it!

**Comment by Siilwyn** [comment]
Published in v7.2.0
